docs/integrations/sources/slack.md

```diff
@@ -49,7 +49,44 @@ The Slack connector should not run into Slack API limitations under normal usage
 
 ### Setup guide
 
-Generate a API access token using the [Slack documentation](https://slack.com/intl/en-ua/help/articles/215770388-Create-and-regenerate-API-tokens)
+{% hint style="info" %}
+If you are using an "legacy" Slack API, skip to the Legacy API Key section below.
+{% endhint %}
+
+In order to pull data out of your Slack instance, you need to create a Slack App. This may sound daunting, but it is actually pretty straight forward. Slack supplies [documentation](https://api.slack.com/start) on how to build apps. Feel free to follow that if you want to do something fancy. We'll describe the steps we followed to creat the Slack App for this tutorial.
```

Comment:
```suggestion
In order to pull data out of your Slack instance, you need to create a Slack App. This may sound daunting, but it is actually pretty straight forward. Slack supplies [documentation](https://api.slack.com/start) on how to build apps. Feel free to follow that if you want to do something fancy. We'll describe the steps we followed to create the Slack App for this tutorial.
```

Is it expected that you mention tutorial in the doc?